### ISSUE TITLE
tmpname - permission denied 

### ISSUE THREAD

**Issue by mnicky** [mkottman/lua-git#13]
There's one `os.tmpname()` call left in [protocol.lua:111](https://github.com/mkottman/lua-git/blob/master/lua/git/protocol.lua#L111) which causes error on my 64bit Windows 7 VM:

```
C:\luadist\> bin\luadist.exe refresh
Downloading repository information...
Error getting manifest: Error when downloading manifest: Error when downloading
the manifest from repository with url: 'git://github.com/LuaDist/Repository.git'
: Error when fetching ref 'refs/heads/master' from git repository 'git://github.
com/LuaDist/Repository.git': ...\bin\..\lib\lua\git\protocol.lua:112: \s26o..pack: Permission denied
```

<br>
Btw, there are also [these](https://github.com/mkottman/lua-git/blob/master/lua/git/pack.lua#L19) [usages](https://github.com/mkottman/lua-git/blob/master/lua/git/pack.lua#L204) of `git.util.tmpfile()` but the function doesn't exist since https://github.com/mkottman/lua-git/commit/044535f57aace1dc8491581da9878fbf8177591f.

**Comment by mkottman** [comment]
Should be fixed in b1ce08a1cdbc5a2fcd9deb5ab4f8c6ebd2ab05d3

**Comment by mnicky** [comment]
After https://github.com/mkottman/lua-git/pull/14 seems fixed. Thanks.
